dbi/dbi.go

```diff
@@ -153,7 +150,6 @@ func (dbiPlg *DbiPlugin) getMetrics() (map[string]interface{}, error) {
 	metrics := map[string]interface{}{}
 
 	err := openDBs(dbiPlg.databases)
-	defer closeDBs(dbiPlg.databases)
```

Comment:
If there was a mislead in drivers name the connection wasn't opened, so we try to close connection which wasn't opened, this causes "unexpected EOF" error. Now we are closing only opened connections.